spec/Signal/Cabin/Infotainment.vspec

```diff
@@ -117,3 +117,48 @@
   max: 250
   unit: m/s
   description: Vehicle speed, as sensed by the GPS receiver.
+
+- HMI:
+  type: branch
+  description: HMI related signals
+
+- HMI.CurrentLanguage:
+  type: string
+  description: ISO 639-1 standard language code for the current HMI
+  sensor: HMI
+
+- HMI.DateFormat:
+  type: string
+  enum: [ "DD MM YYYY", "MM DD YYYY" ]
+  description: Date format used in the current HMI
+  sensor: HMI
+  actuator: HMI
+
+- HMI.TimeFormat:
+  type: string
+  enum: [ "12HR", "24HR" ]
+  description: Time format used in the current HMI
+  sensor: HMI
+  actuator: HMI
+
+- HMI.DistanceUnit:
+  type: string
+  enum: [ "Miles", "Kilometers" ]
```

Comment:
Or use standard abbreviations instead?  (The README even speaks about standard units, so that can be reused)